### PR TITLE
chore: cleanup

### PR DESCRIPTION
**Pull request by kelemeno**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: l1-contracts/scripts/token-migration.ts [hunk 1]
```diff
@@ -218,3 +221,43 @@ async function prepareGovernanceTokenMigrationCall(
     executeCalldata,
   };
 }
+
+const provider = web3Provider();
+
+export async function transferTokensOnForkedNetwork(deployer: Deployer) {
+  // const startToken = 20;
+  // const tokens = tokenList.slice(startToken);
+  // console.log(`From ${startToken}`, tokens);
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/token-migration.ts:230]: please remove commented out code

File: l1-contracts/scripts/verify.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
 // hardhat import should be the first import in the file
 import * as hardhat from "hardhat";
 import { deployedAddressesFromEnv } from "../src.ts/deploy-utils";
-import { getNumberFromEnv, getHashFromEnv, getAddressFromEnv, ethTestConfig } from "../src.ts/utils";
+import { ethTestConfig, getNumberFromEnv, getHashFromEnv, getAddressFromEnv } from "../src.ts/utils";
+// import { ethTestConfig } from "../src.ts/utils";
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/verify.ts:5]: commented out code

File: l1-contracts/scripts/verify.ts [hunk 3]
```diff
@@ -19,6 +20,7 @@ function verifyPromise(address: string, constructorArguments?: Array<any>, libra
         address,
         constructorArguments,
         libraries,
+        // contract:"contracts/bridge/L1ERC20Bridge.sol:L1ERC20Bridge"
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/verify.ts:23]: same

File: l1-contracts/scripts/verify.ts [hunk 4]
```diff
@@ -127,15 +131,27 @@ async function main() {
...
+        genesisBatchHash,
+        genesisIndexRepeatedStorageChanges: genesisRollupLeafIndex,
+        genesisBatchCommitment,
+        diamondCut,
+      },
       protocolVersion,
     },
   ]);
 
+  console.log("initCalldata2", {
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/verify.ts:145]: are you sure we should console log it here?

File: l1-contracts/scripts/verify.ts (hunk 3 above)
Comment (kelemeno) [reply l1-contracts/scripts/verify.ts:23]: We need this to verify the bridge, so I will add a not here and leave it there

File: l1-contracts/scripts/token-migration.ts [hunk 5]
```diff
@@ -218,3 +221,40 @@ async function prepareGovernanceTokenMigrationCall(
     executeCalldata,
   };
 }
+
+const provider = web3Provider();
+
+export async function transferTokensOnForkedNetwork(deployer: Deployer) {
+  const tokenList = ["0x5A520e593F89c908cd2bc27D928bc75913C55C42"];
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/token-migration.ts:228]: please add a comment on how this function is meant to be used and why does it have hardcoded tokenList.

File: l1-contracts/scripts/token-migration.ts [hunk 6]
```diff
@@ -218,3 +221,40 @@ async function prepareGovernanceTokenMigrationCall(
...
+    console.log(`Migrating token ${tokenAddress}`);
+    console.log(
+      `Balance before: ${await erc20contract.balanceOf(deployer.addresses.Bridges.ERC20BridgeProxy)}, ${await erc20contract.balanceOf(deployer.addresses.Bridges.SharedBridgeProxy)}`
+    );
+    await transferTokens(deployer, tokenAddress);
+    console.log(
+      `Balance after: ${await erc20contract.balanceOf(deployer.addresses.Bridges.ERC20BridgeProxy)}, ${await erc20contract.balanceOf(deployer.addresses.Bridges.SharedBridgeProxy)}`
+    );
+  }
+  console.log("From 0", tokenList);
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/token-migration.ts:240]: are you sure we need this log?

File: l1-contracts/scripts/token-migration.ts [hunk 7]
```diff
@@ -218,3 +221,40 @@ async function prepareGovernanceTokenMigrationCall(
...
+  for (const tokenAddress of tokenList) {
+    const erc20contract = IERC20Factory.connect(tokenAddress, provider);
+    if (!(await erc20contract.balanceOf(deployer.addresses.Bridges.ERC20BridgeProxy)).eq(0)) {
+      console.log(`Failed to transfer all tokens ${tokenAddress}`);
+    }
+  }
+}
+
+export async function transferTokens(deployer: Deployer, token: string) {
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/token-migration.ts:249]: same here, explain why do we keep this function here and why its internal params such as "324" are hardcoded

File: l1-contracts/scripts/upgrade-consistency-checker.ts [hunk 8]
```diff
@@ -476,7 +477,7 @@ async function main() {
     await checkIdenticalBytecode(gettersFacet, "GettersFacet");
     await checkIdenticalBytecode(adminFacet, "AdminFacet");
     await checkIdenticalBytecode(bridgeHubImpl, "Bridgehub");
-    await checkIdenticalBytecode(verifier, eraChainId == 324 ? "Verifier" : "TestnetVerifier");
+    await checkIdenticalBytecode(verifier, eraChainId == (324).toString() ? "Verifier" : "TestnetVerifier");
```
Comment (StanislavBreadless) [inline l1-contracts/scripts/upgrade-consistency-checker.ts:480]:
```suggestion
    await checkIdenticalBytecode(verifier, eraChainId == "324" ? "Verifier" : "TestnetVerifier");
```

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**